libs/pq_proto/src/lib.rs

```diff
@@ -0,0 +1,1081 @@
+//! Postgres protocol messages serialization-deserialization. See
```

Comment:
Interesting, that this file was not marked as moved, similar to `sync.rs` below.
Were there any special changes besides moving for this file?

Reply:
A bit, I had to patch some imports and add `mod sync`.